.github/workflows/deploy_prod.yml

```diff
@@ -33,6 +33,8 @@ jobs:
           EXPRESS_SDK_URL: https://dev.cc-embed.adobe.com/sdk/1p/v4/4.1.14/CCEverywhere.js
           SPLUNK_HEC__HEC_TOKEN: ${{ secrets.SPLUNK_HEC__HEC_TOKEN }}
           AIO_LOG_LEVEL: info
+          FT_EARLY_ACCESS: 'SITES-20810'
```

Comment:
Wouldn't it make sense to have this defined as a constant within the code itself? 
Doesn't sound like env variable to me.

Reply:
My reasoning for making this an environment variable is as follows:
1. 'SITES-20810' is referenced both on the frontend and backend.
2. I wanted to keep the source of truth in one single place.
3. Since this is a project with both frontend and backend in the same repo, I thought this might make more sense.